drafts/docker-open-liberty-raspberry-pi.adoc

```diff
@@ -1,60 +1,73 @@
 ---
 layout: post
-title: Create Docker images with Open Liberty compatible with Raspberry Pi 
+title: Open Liberty in Docker on Raspberry Pi
 categories: blog
 author_picture: https://avatars2.githubusercontent.com/u/52958137?s=460&v=4
 author_github: https://github.com/yasmin-aumeeruddy
 blog_description: 
-seo-title: Create Docker images with Open Liberty compatible with Raspberry Pi 
-seo-description: Run Liberty Bikes on the Raspberry Pi using ARM compatible Docker images.
+seo-title: Open Liberty in Docker on Raspberry Pi
+seo-description: 
 ---
-= Create Docker images with Open Liberty compatible with Raspberry Pi 
-It is well known that Raspberry Pi's are useful pieces of technology with lots of potential. We made use of them by deploying a kubernetes cluster to run link:https://github.com/OpenLiberty/liberty-bikes[Liberty Bikes], a real-time web based multiplayer game running on OpenLiberty. Fortunately Open Liberty is small and lightweight so it was never a problem to fit it on the original Raspberry Pi back in 2012. Although, we have used Raspberry Pi 3's for this demonstration.
+= Open Liberty in Docker on Raspberry Pi
 
-The complication when moving to a Raspberry Pi is that it has an ARM CPU at its heart, meaning that machine code compiled for x86 or x86-64 CPUs (the standard CPUs you get in laptops/desktops today) won’t execute. Liberty is written in Java so it doesn’t have to worry about what the CPU architecture is: as long as there is a Java runtime it’s able to run (as the Java runtime itself interprets the Java code and executes it on the hardware). We do have official Open Liberty images on Docker Hub, but they are layered on top of the official IBM image, which is built on top of Ubuntu and doesn’t have a JVM that will run on ARM.
+Yasmin Aumeeruddy <https://github.com/yasmin-aumeeruddy>
+
+It is well known that Raspberry Pi's are useful pieces of technology with lots of potential so I set off on a mission to see how easy it is to get a Raspberry Pi-compatible Docker image of Open Liberty. Fortunately Open Liberty is small and lightweight so it was never a problem to fit it on the original Raspberry Pi back in 2012. Although,I have used Raspberry Pi 3's for this demonstration.
```

Comment:
Can you re-jig the opening paragraph slightly: "Open Liberty is small and lightweight so it was never a problem to run it on the original Raspberry Pi back in 2012. I wanted to see if I could run Open Liberty in a Docker container so I set off on a mission to see how it would be to get a Raspberry Pi-compatible Docker image of Open Liberty. I used a Raspberry Pi 3." or something like that?

Reply:
Then put that same text in the blog description and seo description fields in the front matter of the file too? I think it should be short enough.